app/src/main/java/org/dhis2/data/forms/dataentry/fields/radiobutton/YesNoView.java

```diff
@@ -288,13 +288,13 @@ public void dispatchSetActivated(boolean activated) {
     }
 
     public void setInitialValue(String value) {
-        if (value != null && Boolean.valueOf(value)) {
+        if (value != null && !value.isEmpty() && Boolean.valueOf(value)) {
```

Comment:
I think that this view is not used by the form, only by the datatable. Should this issue only happends in the table?

Reply:
It is used in the forms. It's in the form_yes_no layout and it is returned by the LayoutProvider